release_notes/ocp_3_11_release_notes.adoc

```diff
@@ -223,22 +223,22 @@ See
 xref:../install/configuring_inventory_file.adoc#install-config-configuring-inventory-file[Configuring
 Your Inventory File] for more information.
 
-[[ocp-311-support-for-ansible-2-6]]
-==== Support for Ansible 2.8
+[[ocp-311-support-for-ansible-2-9]]
+==== Support for Ansible 2.9
 
-`openshift-ansible` now requires Ansible 2.8 for both installation of
+`openshift-ansible` now requires Ansible 2.9 for both installation of
 {product-title} 3.11 and upgrading from version 3.10.
 
 The minimum version of Ansible required for {product-title} 3.11 to run
-playbooks is now 2.8.x. On both master and node, use `subscription-manager` to
+playbooks is now 2.9.x. On both master and node, use `subscription-manager` to
```

Comment:
The min version of Ansible we required maybe 2.6? 
`Requires:      ansible >= 2.5.7`
https://github.com/openshift/openshift-ansible/blob/release-3.11/openshift-ansible.spec#L21